.github/actions/process_test_conditions/action.yml

```diff
@@ -0,0 +1,83 @@
+name: 'Process Test Conditions'
+description: 'Process all test-related conditions and inputs to determine final test configuration'
+
+inputs:
+    is-workflow-dispatch:
+        description: 'Whether this is a workflow dispatch event'
+        required: true
+    exclude-slow-dispatch-input-value:
+        description: 'Workflow dispatch input for excluding slow tests'
+        required: true
+    exclude-unstable-dispatch-input-value:
+        description: 'Workflow dispatch input for excluding unstable tests'
+        required: true
+    exclude-release-only-dispatch-input-value:
+        description: 'Workflow dispatch input for excluding release-only tests'
+        required: true
+    fuzz-dispatch-input-value:
+        description: 'Workflow dispatch input for running fuzz tests'
+        required: false
+        default: 'false'
+    link-azle-dispatch-input-value:
+        description: 'Workflow dispatch input for linking to the development version of azle'
+        required: true
+
+outputs:
+    exclude-slow:
+        description: 'Whether to exclude slow tests'
+        value: ${{ steps.process-conditions.outputs.exclude-slow }}
+    exclude-unstable:
+        description: 'Whether to exclude unstable tests'
+        value: ${{ steps.process-conditions.outputs.exclude-unstable }}
+    exclude-release-only:
+        description: 'Whether to exclude release-only tests'
+        value: ${{ steps.process-conditions.outputs.exclude-release-only }}
+    link-azle:
+        description: 'Whether to link to the development version of azle'
+        value: ${{ steps.process-conditions.outputs.link-azle }}
+    fuzz:
+        description: 'Whether to run fuzz tests'
+        value: ${{ steps.process-conditions.outputs.fuzz }}
+
+runs:
+    using: 'composite'
+    steps:
+        - id: set-conditions
```

Comment:
I'm not sure that set-conditions makes sense in this context, let's try "get-conditions" and maybe let's try giving it a more descriptive name

Reply:
let's try determine-workflow-context